metals/src/main/scala/scala/meta/internal/metals/ForwardingMetalsBuildClient.scala

```diff
@@ -23,12 +23,24 @@ import scala.meta.internal.metals.Timer
 import scala.meta.internal.tvp._
 import scala.meta.io.AbsolutePath
 
-import ch.epfl.scala.bsp4j._
 import ch.epfl.scala.{bsp4j => b}
 import com.google.gson.JsonObject
 import org.eclipse.lsp4j.jsonrpc.services.JsonNotification
 import org.eclipse.{lsp4j => l}
 
+/**
+ * Used to forward messages from the build server. Messages might
+ * be mixed if the server is sending messages as well as output from
+ * running. This hasn't been a problem yet, not perfect solution,
+ * but seems to work ok.
```

Comment:
It'll forward all messages from the build server, e.g. if during a running I'll trigger a compilation or some different action, those logs might end up in "program output", right?

Reply:
yes, unfortunately. No way around it for now, we could do filtering I guess but some messages could end up anyway.